components/automate-gateway/api/compliance/reporting/reporting.proto

```diff
@@ -48,6 +48,22 @@ service ReportingService {
    			action: "compliance:reportids:list"
    		};
    	};
+	// should cover /control-items
+	// This api is useful for getting a limited list of control items for latest runs.  It also honors all reporting filters
+	rpc ListControlItems(Query) returns (ControlItems) {
+		option (google.api.http) = {
+   			post: "/compliance/reporting/control-items"
```

Comment:
i'm curious what drove the decision to name the api control-items (vs. controls)

Reply:
nothing.. I can change to controls

Reply:
> nothing.. I can change to controls

would prob be a good post-standup convo for the group (including ux). maybe control-items makes more sense than controls, or vice-versa?  we have a few diff patterns in our api.. 🤷‍♀